pihole/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# CHANGELOG - pihole
+0.0.7 - initial release
+0.0.10 - unit tests added, Service check added, dashboard added, custom tags, note all metrics now tagged with the tartget host.
```

Comment:
No need to have to versions on first merge. First merged version is usually 1.0.0